README.md

```diff
@@ -4,7 +4,11 @@ A JavaScript project for the Leaderboard list app, using webpack and ES6 feature
 
 ## Screenshot
 
-<img src="./src/assets/images/Screenshot 2022-04-13 at 1.16.53 PM.png">
+<img src="./src/assets/images/Screenshot 2022-04-15 at 4.14.54 PM.png">
+
+<!-- ## Live Preview
+
+- Live preview: [click here](https://gnwin.github.io/Leaderboard/) -->
 
 ## Built With
 
```

Comment:
- [ ] README file should contain all that's needed for running the app with the API(s) it consumes. Let's add it

Reply:
github-pages not working properly. css is not rendering

Reply:
css is now rendering properly. gh-pages properly implemented. thanks